cmd/web3/generate.go

```diff
@@ -49,7 +49,7 @@ func GenerateCode(ctx context.Context, c *cli.Context) {
 	bins := []string{c.String("")}
 	types := []string{c.String("pkg")}
 
-	code, err := bind.Bind(types, abis, bins, c.String("pkg"), lang)
+	code, err := bind.Bind(types, abis, bins, nil, c.String("pkg"), lang, nil)
```

Comment:
AFAIK this should work as before with `nil`s. They might be useful here though (e.g. library support all around would be nice).